src/libpisp/backend/pisp_be_config.h

```diff
@@ -30,7 +31,7 @@
 
 #define PISP_BACK_END_NUM_TILES 64
 
-typedef enum {
+enum pisp_be_bayer_enable {
```

Comment:
Not a comment but a question based on my incomplete language knowledge: Does this mean the type names will now be spelled differently in 'C' and C++?

Reply:
Is this because C++ allow you to omit the type category ? (in C++ you can just write `pisp_be_bayer_enable`) ?

Reply:
I don't know - I may be confusing `enum` with `enum class`...

Reply:
I just checked: C++ allows both styles: `pisp_be_bayer_enable` and `enum pisp_be_bayer_enable`. I guess it does not matter for any practical purpose.

Reply:
Yes C++ allows you to omit the type category, but you can still use it if you prefer. Scoped enumerations (`enum class`) in C++ have different implications, but as this is a kernel header and it's plain C, there's no such thing here.

As you said, there shouldn't be any practical differences. In userspace C++ you can omit the type category and still use `pisp_be_bayer_enable` as you used to. In kernel code instead it's usually discouraged to define new type names to omit the category, so you'll have to spell `enum pisp_be_bayer_enable` in full, but I think that's expected.